patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -134,6 +134,15 @@
              float f = 0.5F;
              double d0 = (double)(p_180635_0_.field_73012_v.nextFloat() * 0.5F) + 0.25D;
              double d1 = (double)(p_180635_0_.field_73012_v.nextFloat() * 0.5F) + 0.25D;
+@@ -584,7 +591,7 @@
+         return 0;
+     }
+ 
+-    public float func_149638_a(Entity p_149638_1_)
++    public float func_149638_a(@Nullable Entity p_149638_1_)
```

Comment:
Should this be deprecated in favor of the world, position, and explosion specific version?